lib/state.js

```diff
@@ -569,6 +569,82 @@ class State {
 
     return result.rowCount;
   }
+   
+   /**
+   * Insert an AMI's first usage.
+   */
+  async insertAmiUsage({region, id}, client) {
```

Comment:
Maybe `region` and `id` should not be destructed parameters, as they are non-optional.